src/utils/patchUrlMappings.ts

```diff
@@ -116,6 +116,7 @@ export function patchUrlMappings(
 }
 
 function recursivelyRemapChildNodes(node: Node, mappings: Mapping[]) {
+  attemptSetNodeSrc(node, mappings);
```

Comment:
Super-opinionated-pedantic nit

We should rename the function to `recursivelyRemapNodeAndChildNodes`

or we can call `attemptSetNodeSrc` inside the original mutation's added nodes
```ts
  if (patchSrcAttributes) {
    const callback: MutationCallback = function (mutationsList) {
      for (const mutation of mutationsList) {
        if (mutation.type === 'attributes' && mutation.attributeName === 'src') {
          attemptSetNodeSrc(mutation.target, mappings);
        } else if (mutation.type === 'childList') {
          mutation.addedNodes.forEach((node) => {
            attemptSetNodeSrc(node, mappings);
            recursivelyRemapChildNodes(node, mappings);
          });
        }
      }
    };
```

Either works. I'm pro the code-snippet above as it should result in less function calls. But we should double-check that it still solves the core issue